metamath.c

```diff
@@ -57,7 +56,10 @@
 
 
 
-#define MVERSION "0.198 7-Aug-2021"
+#define MVERSION "0.199.pre 7-Aug-2021"
+/* 0.199.pre
+   30-Dec-2021 mc metamath.c mmdata.c mminou.c mmmaci.c -
+     Remove mmmaci and everything related to THINK_C compiler */

```

Comment:
For future changes, my recommendation is to append them to the `0.199.pre` comment here until the next release. The date is intentionally not updated, to avoid conflicts.